src/types/Client.d.ts

```diff
@@ -13,6 +14,7 @@ export interface ClientConfig {
   keepAlive?: boolean
   headers?: { [key: string]: string | number }
   fetch?: typeof fetch
+  stream?: Subscription
```

Comment:
I think this filed is misplaced. There isn't a `stream` field in the client config object as fas as I know.